mylyn.tasks/org.eclipse.mylyn.tasks.ui/src/org/eclipse/mylyn/internal/tasks/ui/editors/TaskEditorCommentPart.java

```diff
@@ -669,6 +713,16 @@ protected void addActionsToToolbarTitle(ToolBarManager toolBarManager, TaskComme
 			CommentViewer commentViewer) {
 	}
 
+	protected void setTitleImage(ImageHyperlink expandCommentHyperlink, TaskComment taskComment,
+			CommentViewer commentViewer) {
+
+	}
+
+	protected void setUserImage(UserAttributeEditor userImage, TaskAttribute commentAttribute,
```

Comment:
may be a short comment on this method why it is empty? in fact this is a kind of internal API for customization, but empty method looks confusing a bit